### ISSUE TITLE
Tests schlagen fehl

### ISSUE THREAD

**Issue by mayrstefan** [nick81nrw/PVTools#30]
Wenn man die Tests des Frontends ausführt schlagen einige fehl:
```
$ npm run test
```
...
```
Test Suites: 2 failed, 1 skipped, 2 passed, 4 of 5 total
Tests:       15 failed, 12 skipped, 21 passed, 48 total
```
Die problematischen Tests sind:
- read csv file › result has entries
- PV > Consumption › pv generation is more than consumption, battery is loading
- PV > Consumption › pv generation is more than consumption, max battery power load is lower than generation and split in feed in and load
- PV > Consumption › pv generation is more than consumption, max battery power load is lower than generation, only load in battery
- PV > Consumption › pv generation is more than inverter max power generation
- PV > Consumption › pv generation is more than consumption, battery load efficiency diff than unload
- PV > Consumption › pv generation is more than consumption, battery is full, power is feed in
- PV > Consumption › pv generation is more than consumption, battery is full, power is feed in, max feedin power less then generation
- PV > Consumption › pv generation is more than consumption, battery will be fullfilled, diff power is feed in
- PV < Consumption › pv generation is less than consumption, battery is discharging
- PV < Consumption › pv generation is less than consumption, battery is discharging, Batterypower is lower then consumption
- PV < Consumption › pv generation is less than consumption, battery is discharging, Batterypower is lower then consumption, battery will be empty
- PV < Consumption › pv generation is less than consumption, battery unload efficiency is differnt then load
- PV < Consumption › pv generation is less than consumption, battery is empty, cunsumpion from grid
- PV < Consumption › pv generation is less than consumption, battery is nearly empty, diff cunsumpion from grid

Kann es sein, dass die ganzen PV-Tests mit der Änderung der Berechnung zusammenhängen?

**Comment by nick81nrw** [comment]
Ja, die Tests habe ich mi der neuen Berechnung nicht mehr angepasst. Vorher gab es keine "Verteilung von Leistungen innerhalb einer Stunde". Die Regression.json (oder ein subset) Tabelle müsste in die Tests mit aufgenommen werden.
